### PR TITLE
introduced  animations

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 1]
```diff
@@ -29,18 +33,23 @@ function BeerOfTheDayComponent() {
...
-          <Beer
-            id={beer.id}
-            name={beer.name}
-            description={beer.description}
-            abv={beer.abv}
-            img={beer.img}
-            ibu={beer.ibu}
-          />
-        </div>
+      {/* {loading && <Spinner />} */}
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Remove

File: src/screens/BeersFavourite/styles.scss [hunk 2]
```diff
@@ -14,3 +14,17 @@
   margin-right: 80px;
   opacity: 0.4;
 }
+
+.opacity-anims {
```
Comment (mi-chal) [inline src/screens/BeersFavourite/styles.scss]: Why anims?Is it one or few animationS ?

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 3]
```diff
@@ -239,6 +243,19 @@ function SearchBeersComponent() {
     scroolToTop();
   }
 
+  function cleanFilter() {
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: clear is more appropriate word

File: src/screens/SearchBeers/styles.scss [hunk 4]
```diff
@@ -102,3 +102,16 @@
...
+.button-clean-filter {
+  width: 130px;
+  height: 30px;
+  border: 2px solid $myGray;
+  opacity: 0.6;
+  outline: 0;
+  background-color: $mySmokewhite;
+}
+
+.button-clean-filter:hover {
```
Comment (mi-chal) [inline src/screens/SearchBeers/styles.scss]: Use sass here

File: src/shared/components/Counter/styles.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.counter {
+  display: flex;
+  position: absolute;
+  margin-left: 467px;
+  margin-top: 30px;
+  // border-radius: 50%;
```
Comment (mi-chal) [inline src/shared/components/Counter/styles.scss]: Remove

File: src/shared/components/Counter/CounterBeers.jsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React, { useContext } from 'react';
+import './styles.scss';
+import { CounterBeers } from '../../../counterBeer';
+
+function CounterBeer() {
+  const { counter } = useContext(CounterBeers);
+  return <div className="counter">{counter}</div>;
+}
+
+export default CounterBeer;
```
Comment (mi-chal) [inline src/shared/components/Counter/CounterBeers.jsx]: Remove everything that is connected with counter and change PR name.

File: src/shared/components/FavouriteButton/styles.scss [hunk 7]
```diff
@@ -14,3 +14,18 @@
 .icon:hover {
   opacity: 0.6;
 }
+
+.anim {
```
Comment (mi-chal) [inline src/shared/components/FavouriteButton/styles.scss]: Would be awesome to name this animation better by e.g. what it does.

Comment (mi-chal) [reply src/shared/components/FavouriteButton/styles.scss]: Like you did with `showInFromNone`

File: src/screens/SearchBeers/styles.scss (hunk 4 above)
Comment (mi-chal) [reply src/screens/SearchBeers/styles.scss]: You should also keep pattern of buttons that are already in the app. All buttons have border-radius and background - this one does not.

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

File: src/screens/BeersFavourite/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/screens/BeersFavourite/styles.scss]: Done

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 3 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done

File: src/shared/components/Counter/styles.scss (hunk 5 above)
Comment (dwantula) [reply src/shared/components/Counter/styles.scss]: Done

File: src/shared/components/Counter/CounterBeers.jsx (hunk 6 above)
Comment (dwantula) [reply src/shared/components/Counter/CounterBeers.jsx]: Done

File: src/shared/components/FavouriteButton/styles.scss (hunk 7 above)
Comment (dwantula) [reply src/shared/components/FavouriteButton/styles.scss]: Done

**Pull request merged**